File: python/jzoffer-10.py
# -*- coding:utf-8 -*-
class Solution:
    def rectCover(self, number):
        # write code here
        if number == 0:
            return 0
        if number == 1:
            return 1
        if number == 2:
            return 2
        x,a,b = 0,0,1
        while x < number:
            a,b = b,a+b
            x += 1
        return b

File: java/jzoffer-10.java
public class Solution {
    public int RectCover(int target) {
        if(target == 0){
            return 0;
        }
        if(target == 1){
            return 1;
        }
        if(target == 2){
            return 2;
        }
        int x = 0,a = 0,b = 1;
        int i = 0;
        int j = 0;
        while(x < target){
            i = b;
            j = a + b;
            a = i;
            b = j;
            x += 1;
        }
        return b;
    }
}